### PR TITLE
Aws patchbaseline

### PR DESCRIPTION
**Pull request by pkshah1006**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: aws-ssm-patchbaseline/src/main/java/software/amazon/ssm/patchbaseline/ListHandler.java [hunk 1]
```diff
@@ -12,27 +12,19 @@
...
 import software.amazon.ssm.patchbaseline.utils.SsmClientBuilder;
 import software.amazon.awssdk.services.ssm.model.SsmException;
-
+import software.amazon.ssm.patchbaseline.translator.resourcemodel.ListResourceModelTranslator;
+import static software.amazon.ssm.patchbaseline.ResourceModel.TYPE_NAME;
 import java.util.ArrayList;
 import java.util.List;
 import java.util.stream.Collectors;
 import com.google.common.annotations.VisibleForTesting;
+import java.io.File;
```
Comment (dandaji) [inline aws-ssm-patchbaseline/src/main/java/software/amazon/ssm/patchbaseline/ListHandler.java:21]: this is not used

File: aws-ssm-patchbaseline/src/main/java/software/amazon/ssm/patchbaseline/ListHandler.java [hunk 2]
```diff
@@ -12,27 +12,19 @@
...
-
-    public ListHandler() {
-        readHandler = new ReadHandler();
-    }
-
-    @VisibleForTesting
-    void setReadHandler(final ReadHandler readHandler) {
-        this.readHandler = readHandler;
-    }
+    protected static final String PATCH_BASELINE_RESOURCE_NAME = "PatchBaseline";
```
Comment (dandaji) [inline aws-ssm-patchbaseline/src/main/java/software/amazon/ssm/patchbaseline/ListHandler.java:27]: this is not used

File: aws-ssm-patchbaseline/src/main/java/software/amazon/ssm/patchbaseline/translator/resourcemodel/ListResourceModelTranslator.java [hunk 3]
```diff
@@ -0,0 +1,40 @@
+package software.amazon.ssm.patchbaseline.translator.resourcemodel;
+
+import software.amazon.ssm.patchbaseline.ResourceModel;
+import software.amazon.ssm.patchbaseline.utils.SimpleTypeValidator;
+import software.amazon.awssdk.services.ssm.model.PatchBaselineIdentity;
+import software.amazon.awssdk.services.ssm.model.DescribePatchBaselinesResponse;
+
+import java.util.ArrayList;
```
Comment (dandaji) [inline aws-ssm-patchbaseline/src/main/java/software/amazon/ssm/patchbaseline/translator/resourcemodel/ListResourceModelTranslator.java:8]: is ArrayList used?

File: aws-ssm-patchbaseline/src/main/java/software/amazon/ssm/patchbaseline/TagHelper.java [hunk 4]
```diff
@@ -67,9 +67,9 @@
                     .stream()
                     .collect(Collectors.mapping(entry ->
                                     Tag.builder()
-                                       .key(entry.getKey())
-                                       .value(entry.getValue())
-                                       .build(),
+                                            .key(entry.getKey())
```
Comment (dandaji) [inline aws-ssm-patchbaseline/src/main/java/software/amazon/ssm/patchbaseline/TagHelper.java:70]: this file is not changed

File: aws-ssm-patchbaseline/inputs/inputs_1_read.json [hunk 5]
```diff
@@ -0,0 +1,48 @@
+{
```
Comment (dandaji) [inline aws-ssm-patchbaseline/inputs/inputs_1_read.json:1]: is this used in the test?

File: aws-ssm-patchbaseline/.gitignore [hunk 6]
```diff
@@ -1,20 +1,16 @@
...
 *.iml
 .idea
 out.java
 out/
 .settings
 .project
-
 # auto-generated files
 target/
-
```
Comment (dandaji) [inline aws-ssm-patchbaseline/.gitignore:18]: this file is not changed

File: aws-ssm-patchbaseline/src/main/java/software/amazon/ssm/patchbaseline/ReadHandler.java [hunk 7]
```diff
@@ -15,6 +12,8 @@
 import software.amazon.ssm.patchbaseline.translator.resourcemodel.ReadResourceModelTranslator;
 import static software.amazon.ssm.patchbaseline.ResourceModel.TYPE_NAME;
 import software.amazon.ssm.patchbaseline.utils.SsmClientBuilder;
+import software.amazon.cloudformation.proxy.AmazonWebServicesClientProxy;
```
[the remark this answers is not in the document]
Comment (pkshah1006) [reply aws-ssm-patchbaseline/src/main/java/software/amazon/ssm/patchbaseline/ReadHandler.java:15]: We didnot do any changes in this file and but we have imported this AmazonWebServicesClientProxy class for handleRequest method. This method is using this as parameter.
 Please find the method template as below
@Override
    public ProgressEvent<ResourceModel, CallbackContext> handleRequest(
        final AmazonWebServicesClientProxy proxy,
        final ResourceHandlerRequest<ResourceModel> request,
        final CallbackContext callbackContext,
        final Logger logger) {
}

File: aws-ssm-patchbaseline/src/main/java/software/amazon/ssm/patchbaseline/ListHandler.java (hunk 1 above)
Comment (pkshah1006) [reply aws-ssm-patchbaseline/src/main/java/software/amazon/ssm/patchbaseline/ListHandler.java]: Removed the unused import statement

File: aws-ssm-patchbaseline/src/main/java/software/amazon/ssm/patchbaseline/translator/resourcemodel/ListResourceModelTranslator.java (hunk 3 above)
Comment (pkshah1006) [reply aws-ssm-patchbaseline/src/main/java/software/amazon/ssm/patchbaseline/translator/resourcemodel/ListResourceModelTranslator.java]: Removed the unused import statement

File: aws-ssm-patchbaseline/src/main/java/software/amazon/ssm/patchbaseline/TagHelper.java (hunk 4 above)
Comment (pkshah1006) [reply aws-ssm-patchbaseline/src/main/java/software/amazon/ssm/patchbaseline/TagHelper.java:70]: yes this file has no changes, but i have formate the full file so it was showing this changes as alignment problem. 
No code changes are made on this file

File: aws-ssm-patchbaseline/inputs/inputs_1_read.json (hunk 5 above)
Comment (pkshah1006) [reply aws-ssm-patchbaseline/inputs/inputs_1_read.json]: Actually i have used this this files for CFN test case.. The test case process also not using this json files for testing payload. Now i have removed the files

File: aws-ssm-patchbaseline/inputs/inputs_1_list.json [hunk 8]
```diff
@@ -0,0 +1,53 @@
+{
```
[the remark this answers is not in the document]
Comment (pkshah1006) [reply aws-ssm-patchbaseline/inputs/inputs_1_list.json]: Actually i have used this this files for CFN test case.. The test case process also not using this json files for testing payload. Now i have removed the files

File: aws-ssm-patchbaseline/inputs/inputs_1_delete.json [hunk 9]
```diff
@@ -0,0 +1,3 @@
+{
```
[the remark this answers is not in the document]
Comment (pkshah1006) [reply aws-ssm-patchbaseline/inputs/inputs_1_delete.json]: Actually i have used this this files for CFN test case.. The test case process also not using this json files for testing payload. Now i have removed the files

**Pull request closed without merging**